Bottom: eb072e942d2472dea5eb32837e3ff88306673b3d
Top:    ad7b8f87ee49afacec4583276073752213bdb533
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-19 00:15:55 +0000

Refresh of try-bert-s-dxf-hid

---

diff --git a/src/hid/dxf/dxf.c b/src/hid/dxf/dxf.c
index 2c7ca29..a3338b2 100644
--- a/src/hid/dxf/dxf.c
+++ b/src/hid/dxf/dxf.c
@@ -290,7 +290,7 @@ RCSID ("$Id$");
 /*!
  * \brief Debugging on/off switch.
  */
-#define DEBUG 1
+//#define DEBUG 1
 
 /*!
  * \brief Error handling of not yet implemented functions.
